lib/CHeroHandler.h

```diff
@@ -122,9 +122,9 @@ class DLL_LINKAGE CHero
 		h & iconSpecLarge;
 		h & portraitSmall;
 		h & portraitLarge;
-		h & battleImage;
 		if(version >= 759)
 		{
+			h & battleImage;

```

Comment:
Correct is:
```
	if(version >= 790)
	{
		h & battleImage;
	}
```

after old code

Reply:
Thought the 'new handle format' was implemented after 759. But know i understand, we dont have a massive change on format, just a ponctual ones based on version. Thanks for that clarify